contracts/VotingMachines/Governance.sol

```diff
@@ -0,0 +1,509 @@
+pragma solidity ^0.4.18;
+
+import "../controller/Reputation.sol";
+import "./IntVoteInterface.sol";
+import "../universalSchemes/UniversalScheme.sol";
+import "./GovernanceFormulasInterface.sol";
+
+
+
+contract Governance is IntVoteInterface,UniversalScheme,GovernanceFormulasInterface {
```

Comment:
Should this have comments describing basically what Governance does?

Reply:
Update at a6ba131 .
Though still need more detail doc/description.